playbooks/support/responding.md

```diff
@@ -11,6 +11,23 @@ respond to these issues.
   These automated system alerts _may_ be the first sign of a problem but don't _necessarily_ warrant treatment as
   immediate incidents. E.g., load-balancers may route around failing EC2 instances or auto-scaling may bring up
   capacity in response to load.
+- Monitor and acknowledge automated uptime incidents created by OpsGenie.
+
+  - For high-criticality systems, OpsGenie will open incidents if
+    [Pingdom](https://my.pingdom.com/app/newchecks/checks) uptime checks are failing. Alerts will automatically
+    close if resolved.
```

Comment:
What do you think of clarifying this behavior to something like:

Alerts will automatically close within OpsGenie if resolved via Pingdom.